ivadomed/scripts/prepare_dataset_vertebral_labeling.py

```diff
@@ -87,35 +109,36 @@ def extract_mid_slice_and_convert_coordinates_to_heatmaps(path, suffix, aim=-1):
             heatmap = imed_maths.heatmap_generation(label_array[:, :], 10)
             arr_pred_ref_space = imed_loader_utils.reorient_image(np.expand_dims(heatmap[:, :], axis=0), 2, lab, nib_ref_can)
             nib_pred = nib.Nifti1Image(arr_pred_ref_space, lab.affine)
-            nib.save(nib_pred, os.path.join(path, 'derivatives', 'labels', t[i], 'anat', t[i] + suffix +
-                                            '_mid_heatmap' + str(aim) + '.nii.gz'))
+            nib_pred = rotate_nifti(nib_pred)
+            nib.save(nib_pred, os.path.join(path, 'derivatives', 'labels', t[i], 'anat', t[i]+'_rec-mid' +suffix+ '_heatmap'+str(aim)
+                                            + '.nii.gz'))
         else:
             pass
 
 
+
 def get_parser():
     parser = argparse.ArgumentParser()
-    parser.add_argument("-p", "--path", dest="path", required=True, type=str,
-                        help="Path to bids folder",
-                        metavar=imed_utils.Metavar.file)
-    parser.add_argument("-s", "--suffix", dest="suffix", required=True, type=str,
-                        help="""Suffix of the input file as in
-                                sub-xxxSUFFIX.nii.gz (E.g., _T2w)""",
-                        metavar=imed_utils.Metavar.str)
-    parser.add_argument("-a", "--aim", dest="aim", default=-1, type=int,
-                        help="""-1 or positive int. If set to any positive int,
-                                only label with this value will be taken into account""",
-                        metavar=imed_utils.Metavar.int)
+    parser.add_argument("-p", "--path", dest="path", required=False, type=str,default="/data/data/data-multi-subject/",
+                        help="Path to bids folder")
```

Comment:
i would keep it mandatory and not use "/data/data/data-multi-subject/" as default because we cannot assume every user will have this path

Reply:
Default values are removed.